### PR TITLE
Handle multiple parquet RowGroup/Column

### PR DESCRIPTION
**Pull request by congqixia**
Related to #18968

/kind bug

Signed-off-by: Congqi Xia <congqi.xia@zilliz.com>

### CODE REVIEW

File: internal/storage/payload_reader.go [hunk 1]
```diff
@@ -80,18 +80,27 @@ func (r *PayloadReader) GetBoolFromPayload() ([]bool, error) {
...
 
+	var offset int
 	values := make([]bool, r.numRows)
-	total, valuesRead, err := reader.ReadBatch(r.numRows, values, nil, nil)
-	if err != nil {
-		return nil, err
+	for i := 0; i < r.reader.NumRowGroups(); i++ {
+		rg := r.reader.RowGroup(i)
+		for j := 0; j < rg.NumColumns(); j++ {
+			reader, ok := rg.Column(j).(*file.BooleanColumnChunkReader)
```
Comment (xiaofan-luan) [inline internal/storage/payload_reader.go:89]: you can not iterate through all columns because there is no guarantee that they are the same data type

Comment (congqixia) [reply internal/storage/payload_reader.go]: Fixed

**Pull request merged**